extensions/2.0/Khronos/KHR_audio_graph/README.md

```diff
@@ -0,0 +1,1279 @@
+   <td>number
+   </td>
+   <td>gain applied to the signal by the emitter.
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>spatial properties
+   </td>
+   <td>object
+   </td>
+   <td>See 5.2 Spatial properties.
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+
+### 5.2 Spatial properties
+
+
+<table>
+  <tr>
+   <td>
+   </td>
+   <td><strong>Type</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+   <td><strong>Required</strong>
+   </td>
+   <td><strong>Notes</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>spatialization model
+   </td>
+   <td>string
+   </td>
+   <td>Determines which spatialization model will be used to position the audio in 3D space (equal power, HRTF, Custom).
+<p>
+equalpower: Represents the equal-power panning algorithm, generally regarded as simple and efficient. equalpower is the default value.
+<p>
+HRTF: Renders a stereo output of higher quality than equalpower — it uses a convolution with measured impulse responses from human subjects.
+<p>
+Custom: User-defined panning algorithm.
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>attenuation
+   </td>
+   <td>object
+   </td>
+   <td>See 5.3 Attenuation properties.
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>shape
+   </td>
+   <td>object
+   </td>
+   <td>See 5.4 Shape properties
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+
+### 5.3 Attenuation properties
+
+
+<table>
+  <tr>
+   <td>
+   </td>
+   <td><strong>Type</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+   <td><strong>Required</strong>
+   </td>
+   <td><strong>Notes</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>distance model
+   </td>
+   <td>string
+   </td>
+   <td>Specifies the distance model for the audio emitter  linear, inverse, exponential, custom).
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>ref distance
+   </td>
+   <td>number
+   </td>
+   <td>A reference distance for reducing volume as the emitter moves further from the listener.
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>max distance
+   </td>
+   <td>number
+   </td>
+   <td>The maximum distance between the emitter and listener, beyond which the audio cannot be heard.
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>rolloff factor
+   </td>
+   <td>number
+   </td>
+   <td>Describes how quickly the volume is reduced as the emitter moves away from the listener.
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+
+### 5.4 Shape properties
+
+
+<table>
+  <tr>
+   <td>
+   </td>
+   <td><strong>Type</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+   <td><strong>Required</strong>
+   </td>
+   <td><strong>Notes</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>type
+   </td>
+   <td>string
+   </td>
+   <td>Shape in which emitter emits audio (cone, omnidirectional, custom).
```

Comment:
Why is `custom` included here? Is there a way of defining a custom shape? How would it be implemented?

Reply:
It would be implemented by another glTF extension, extending this extension.